### ISSUE TITLE
PWABuilder generates invalid badge images for Windows 10

### ISSUE THREAD

**Issue by JudahGabriel** [pwa-builder/ManifoldJS#289]
PWABuilder generates a set of images for Windows 10 based on the images in my manifest.json file in my web app.

If I have a 48x48 image or a 96x96 image in my manifest, PWABuilder generates an appx that uses those images for BadgeLogo.scale-200.png and BadgeLogo.scale-400.png.

The problem is, **this will fail when submitted to the Windows Store** with the following automated error:

> Image reference "images\BadgeLogo.png": The image "images\BadgeLogo.scale-200.png" has an ABGR value "0xFF583F33" at position (0, 0) that is not valid. The pixel must be white (##FFFFFF) or transparent (00######).

BadgeLogo must be an image containing only white pixels or transparent pixels.

Bottom line: PWA Builder generates an .appx that will fail store submission (and fail the AppX Resource Validator) due to generating invalid badge images. **There should be a way to opt-out of generating badges.** Alternately, PWA Builder should create badge images that are comprised only of white or transparent pixels.

**Comment by boyofgreen** [comment]
Hello @JudahGabriel  We have gone through a number of different iterations to make this work, and in the end we decided to not build out badge images at all.  They are only used for a scenario where you are showing badges on the lock screen, and we figured if a dev was intending to support that scenario they could build out a badge image for it as well.  The problem stemmed from the badge image being the same size as the smallest iOS image.
  We have this fix in the preview server already (preview.pwabuilder.com) and hope to push it out to prod later this week or early next week.  In the mean time you should be able to generate with the preview site even with those image sizes in the manifest.  Let me know if you run into any issues there.
Thanks,
Jeff

**Comment by JudahGabriel** [comment]
Great. I wasn't aware of the preview server. I'll give that a shot.

p.s. this is an awesome tool! You guys should get together with the Azure team and build [this killer feature](https://twitter.com/JudahGabriel/status/966061715539587072). I'd pay to have that.

**Comment by boyofgreen** [comment]
Thanks for the feedback too.  I need to get that pushed out to master.  So as far as your killer feature, I actually looked into doing this for each store, but there was a lot of question on "publishing on behalf" of another dev.  That being said, I think there is a lot that can be done to help a developer get right to the point of your killer feature.

**Comment by JudahGabriel** [comment]
FWIW, I [suggested this on Azure Feedback](https://feedback.azure.com/forums/223579-azure-portal/suggestions/33394270-you-published-a-pwa-push-to-app-stores).
